frontend/src/components/page_content/page-content-template.js

```diff
@@ -45,17 +53,23 @@ class PageContentTemplate extends React.Component {
   }
 
   setFactorsSaveCancel() {
+    this.saveLabel = "Next";
```

Comment:
i think this should be set in the constructor cuz it always has to render as something, so you can be sure its set by putting it there instead